MoreEffectiveTransfer.csproj

```diff
@@ -88,16 +86,19 @@
     </Content>
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
+    <PackageReference Include="Lib.Harmony">
+      <Version>1.2.0.1</Version>
+    </PackageReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-
   <!-- Copy all build outputs to mods folder using standard MSBuild Copy task. -->
   <Target Name="AfterBuild">
     <ItemGroup>
-      <OutputFiles Include="$(OutDir)**\*"/>
-    </ItemGroup>  
-    <Message Text="Ensuring @(OutputFiles) are in $(CitiesSkylinesModsPath)$(SolutionName)\%(RecursiveDir)" Importance="high"/>
-    <Copy SourceFiles="@(OutputFiles)" DestinationFolder="$(CitiesSkylinesModsPath)$(SolutionName)\%(RecursiveDir)" ContinueOnError="false" OverwriteReadOnlyFiles="true"/>
+      <OutputFiles Include="$(OutDir)**\*" />
+    </ItemGroup>
+    <!-- <Message Text="Ensuring @(OutputFiles) are in $(CitiesSkylinesModsPath)$(SolutionName)\%(RecursiveDir)" Importance="high" />-->
+    <!-- <Copy SourceFiles="@(OutputFiles)" DestinationFolder="$(CitiesSkylinesModsPath)$(SolutionName)\%(RecursiveDir)" ContinueOnError="false" OverwriteReadOnlyFiles="true" />-->
+    <Message Text="Ensuring @(OutputFiles) are in $(CitiesSkylinesModsPath)\%(RecursiveDir)" Importance="high" />
+    <Copy SourceFiles="@(OutputFiles)" DestinationFolder="$(CitiesSkylinesModsPath)\%(RecursiveDir)" ContinueOnError="false" OverwriteReadOnlyFiles="true" />
```

Comment:
Use `"$(CitiesSkylinesModsPath)%(RecursiveDir)"` instead of "$(CitiesSkylinesModsPath)\%(RecursiveDir)" to avoid duplicating the backslash at the end of the first variable.